centralstore/smartwrapper.go

```diff
@@ -157,7 +157,10 @@ func (w *SmartWrapper) manageTimeouts(ctx context.Context, timeout time.Duration
 	})
 	defer span.End()
 
-	st, err := w.BasicStore.GetTracesForState(ctx, fromState)
+	// process up to 20% of the channel size
+	n := w.Config.GetCentralStoreOptions().SpanChannelSize * 20 / 100
```

Comment:
I'm not sure if this is the right math but I don't think this value should be a user configured value

Reply:
I don't think this is the value we want to use. We'd like to grab a fraction of all the traces stored; the size of the input channel doesn't really affect how many we have stored. If we make this configurable we could play with it, but if we don't want a config let's try a number like, say, 400.

Reply:
Since we do get the metrics of the total keys in a given state from redis, maybe we can use that to calculate the size here?

Reply:
I made it to be configurable for now

Reply:
OK; I think that's a better choice. We do use metrics in a load-bearing way for Stress Relief, but I feel differently about that because its job is to monitor the state of the system, and that's also what metrics do.